proxy-node-shared/src/main/java/uk/gov/ida/notification/contracts/HubResponseTranslatorRequest.java

```diff
@@ -39,7 +39,7 @@
 
     @NotNull
     @JsonProperty
-    private URI eidasIssuerEntityId;
+    private URI connectorEntityId;
```

Comment:
Is this the `connectorEntityId` for the sending EU country? Maybe this could be something more specific to differentiate between their connector and ours. I know in this context it could only be theirs but it did make me think for a few seconds.

Reply:
My thought was this is the Proxy Node repo, so "connector" would always be the connecting country.

Reply:
Yep, it's not really an issue. I can live live with it!